app/assets/javascripts/admin/addon/templates/customize-themes-show.hbs

```diff
@@ -1,4 +1,5 @@
 <div class="show-current-style">
+  {{plugin-outlet name="admin-customize-themes-show-top" args=(hash remote=model.remote_theme)}}
```

Comment:
Can we pass the `theme` object here, rather than the `remote_theme`? I realise we only need `remote_theme` for our intended use case, but if we use `theme` then the outlet might be useful for other themes/plugins in future?